types/leaflet/index.d.ts

```diff
@@ -1297,13 +1321,13 @@ export class GeoJSON<P = any> extends FeatureGroup<P> {
     /**
      * Adds a GeoJSON object to the layer.
      */
-    addData(data: geojson.GeoJsonObject): Layer;
+    addData(data: geojson.GeoJsonObject): this;
 
     /**
      * Resets the given vector layer's style to the original GeoJSON style,
      * useful for resetting style after hover events.
      */
-    resetStyle(layer?: Layer): Layer;
+    resetStyle(layer?: Layer): this;
```

Comment:
I'm not sure if a `this` return type is useful here (or at all). What is the reasoning behind this change?